src/controllers/events.js

```diff
@@ -180,9 +180,28 @@ async function joinedVolunteers(req, res) {
     }
 }
 
+async function unjoinedVolunteers(req, res) {
```

Comment:
function name should be a verb of smth the function do `unjoinedVolunteers` sounds like a variable name
maybe we can change it to, `removeVolunteers` `unjoinVolunteers`

Reply:
done. I refactor the code based on @muhammed-shihebi advice